electron/main.ts

```diff
@@ -471,8 +471,22 @@ ipcMain.on('frontendReady', () => {
 })
 
 // Maybe this can help with white screens
```

Comment:
What white screens are being mentioned here?

Reply:
I think was an old Electron bug that sometimes the app became blank in some setups, this was a general issue not only in Heroic so I found this solution when searching on the web.
Not sure if they fixed it already.